File: test/demo/wslc-polyglot-hello/polyglot/servers/srv.jl
using Sockets

port = parse(Int, ARGS[1])

srv = listen(IPv4(0), port)
conn = accept(srv)
readavailable(conn)                            # Windows request
write(conn, "hello world from [julia]\n")
readavailable(conn)                            # Windows ack -> exit
close(conn)
close(srv)

File: test/demo/wslc-polyglot-hello/polyglot/servers/srv.dart
import 'dart:io';

void main(List<String> a) async {
  final port = int.parse(a[0]);

  final server = await ServerSocket.bind('0.0.0.0', port);

  await for (final conn in server) {
    conn.listen((d) {                                  // Windows request
      conn.write('hello world from [dart]\n');
    });
    await Future.delayed(Duration(milliseconds: 800)); // Windows ack -> exit
    conn.destroy();
    break;
  }

  await server.close();
}
